includes/admin_import.php

```diff
@@ -410,6 +410,13 @@ function tmsl_import_page() {
 		delete_option('tsml_mapbox_key');
 	}
 
+	//change geocoding method
```

Comment:
In experimenting with this, we should probably validate the choice. For example, I don't have a Google key defined, but was still able to select the option of "Use my Google API key". This should probably alert than a Google API key is needed, and then revert to the previous value?